src/rabbit_reader.erl

```diff
@@ -191,7 +191,7 @@ init(Parent, HelperSup, Ref) ->
 system_continue(Parent, Deb, {Buf, BufLen, State}) ->
     mainloop(Deb, Buf, BufLen, State#v1{parent = Parent}).
 
--spec system_terminate(_,_,_,_) -> none().
+-spec system_terminate(_,_,_,_) -> no_return().
```

Comment:
`none()` and `no_return()` should be equivalent. But somehow only `no_return()` fixes the Dialyzer warning. If anyone has an explication...